lib/power_api/generator_helper/template_builder_helper.rb

```diff
@@ -5,6 +5,12 @@ def concat_tpl_statements(*methods)
     methods.reject(&:blank?).join("\n")
   end
 
+  def conditional_code(condition)
+    return unless condition
+
+    yield
+  end
```

Comment:
Check the place where I'm using this. I think it's read better than inline conditionals.
https://github.com/platanus/power_api/pull/38/files/8985dc8cf84d55574a63f9f86a74e3280a5b92a4..8da91fa00ed1a5f5c528f024763795da7c5e9a2d#diff-a95ac771728044ca29abe982d42c335a9f6a17605c915ab1bd38dc4292563a9bR210

Reply:
But in that place I think it would be the same as:

```
    concat_tpl_statements(
      "before do",
        if auth && authenticated_resource? do
          "sign_in(#{authenticated_resource.snake_case})"
        end,
        "perform",
      "end\n"
    )
```

Which reads pretty similar. I think having a wrapper around an `if` can add more confusion than benefit, but if you think it's worth it for the readability, then let's keep it

Reply:
you're right, i'll change it.

Reply:
mmm nope, can't do this between parameters

![image](https://user-images.githubusercontent.com/3026413/149401941-58d937b4-a084-4a7a-bc58-1ec2414d7330.png)

Can use inline conditions but I prefer the method.
![image](https://user-images.githubusercontent.com/3026413/149402094-81a76ac2-9cac-4439-bd3a-6861f848a341.png)

Reply:
it's failing cause the of the extra do in the `if` line

Reply:
changed